src/main/java/com/google/cloud/genomics/dataflow/pipelines/VariantSimilarity.java

```diff
@@ -46,21 +55,29 @@
  */
 public class VariantSimilarity {
   private static final String VARIANT_FIELDS
-      = "nextPageToken,variants(start,calls(genotype,callSetName))";
+    = "nextPageToken,variants(start,calls(genotype,callSetName))";
+
+  public static interface VariantSimilarityOptions extends GenomicsDatasetOptions, GCSOptions {
+    @Default.Boolean(true)
+    Boolean getUseAlpn();
```

Comment:
I can't find where this new option is used. Perhaps add a comment indicating what it means?